addons/account/views/account_invoice_view.xml

```diff
@@ -594,19 +593,20 @@
                         <page string="Other Info" name="other_info">
                             <group>
                                 <group>
+                                    <field name="user_id" groups="base.group_user"/>
```

Comment:
Why the group ? I think you have to be at least employee to have access to backend form view.

Reply:
Yes, I agree with you(@jem-odoo ) but I have just changed fields position. the group already existed on the fields 

if you want to remove it then I will do?

Reply:
yes, while we are at it, we can remove it ;)

Reply:
Removed.